x/rollup/keeper/msg_server.go

```diff
@@ -10,18 +10,6 @@ import (
 	"github.com/samber/lo"
 )
 
-type msgServer struct {
-	*Keeper
-}
-
-// NewMsgServerImpl returns an implementation of the MsgServer interface
-// for the provided Keeper.
-func NewMsgServerImpl(keeper *Keeper) types.MsgServer {
-	return &msgServer{Keeper: keeper}
-}
-
-var _ types.MsgServer = msgServer{}
```

Comment:
We may still want to leave in the compile time interface check here, it seems like a common pattern for cosmos sdk msg servers.

`var _ types.MsgServer = &Keeper{}`

Reply:
[Add interface check](https://github.com/polymerdao/monomer/pull/179/commits/6aefe17933ec159ec455986946a5f07b89cf3799)